libs/payments/stripe/src/lib/stripe.client.ts

```diff
@@ -28,6 +28,17 @@ export class StripeClient {
     return this.stripe.customers.retrieve(customerId);
   }
 
+  /**
+   * Updates customer record with the values of the parameters passed
+   *
+   * @param customerId The Stripe customer ID of the customer to update
+   * @param params Values to be updated in customer record
+   * @returns
+   */
+  async updateCustomer(customerId: string, params: any) {
```

Comment:
The params argument should be typed similar to the method below this, otherwise you can pass any value in for the second arg.

Reply:
Updated.